ard/viz/layout.py

```diff
@@ -122,6 +123,34 @@ def plot_layout(
             landscape=False,
         )
 
+    if include_mooring_system:
```

Comment:
Have you successfully run this with cable and mooring? I think we may need to have the mooring plot last.